openapi/kas-fleet-manager.yaml

```diff
@@ -256,6 +256,81 @@ paths:
                   $ref: '#/components/examples/500Example'
     parameters:
       - $ref: "#/components/parameters/id"
+  /api/kafkas_mgmt/v1/kafkas/{id}/promote:
```

Comment:
love the clarity that this new endpoint sends! Kudos @miguelsorianod @ziccardi @JameelB

Reply:
I updated this PR description to give more details about this PR functionality and design decisions about it